Userland/Libraries/LibVideo/VP9/Decoder.h

```diff
@@ -36,6 +36,10 @@ class Decoder : public VideoDecoder {
 private:
     typedef i32 Intermediate;
 
+    static constexpr size_t BLOCK_MAXIMUM_DIMENSIONS = 64ULL;
+    static constexpr size_t BLOCK_MAXIMUM_SIZE = BLOCK_MAXIMUM_DIMENSIONS * BLOCK_MAXIMUM_DIMENSIONS;
+    static constexpr size_t TRANSFORM_MAXIMUM_SIZE = 32ULL * 32ULL;
```

Comment:
These should have MAXIMUM at the beginning to make them correct words.

Reply:
WHY ARE THESE SCREAMING AT ME THATS ONLY RESERVED FOR MACROS

Reply:
Fixed both of those, sorry for your eardrums lol